quick_start/my_text_classifier_bert.jsonnet

```diff
@@ -0,0 +1,42 @@
+local bert_model = "bert-base-uncased";
```

Comment:
@matt-gardner This is the BERT config files I'm going to use for the "next steps" chapter. There's this thing called `bert_pooler` (a `Seq2VecEncoder `) which I've never used before and am not sure how it fits with the rest of the pipeline.

Reply:
As in, I know you have to pool the embedding for the `[CLS]` token but what should we use as the TokenEmbedder if we used BertPooler as a Seq2VecEncoder?

Reply:
It was my misunderstanding. I just realized you simply need to combine BertEmbedder and BertPooler. But as it is, you need to modify both the embedder and the encoder to support BERT, and add some cryptic fields (`allow_unmatched_keys` and `embedder_to_indexer_map`). The point of the "next steps" chapter is to show how easy it is to swap in a different contextualizer but this could have an opposite effect. I'm starting to wonder if this should make it to the Part 2 at all. @matt-gardner

Reply:
So you think it's ok to show this config and briefly (only briefly) talk about how readers should go about using pretrained contextualizers? I'm talking about here https://github.com/allenai/allennlp-course/blame/master/chapters/next-steps.md#L21

Reply:
I think so - it's just changing a few things (all of the parts you would expect) in the config.  When the crazy parameters are removed, it should be a pretty simple thing to show, with a pointer to a more in-depth discussion elsewhere.

Reply:
Do you disagree?

Reply:
No, I think we can show this. "How to use BERT" is one of the most commonly asked questions about AllenNLP, so we can be upfront about this.

Do you think this PR is good to merge? I also added a requirements file for binder. When this gets merged we can point to this repo so that binder can do imports properly.